README.md

```diff
@@ -2,15 +2,48 @@
 
 We want the [Flux project](https://github.com/fluxcd) to be the vendor-neutral home for GitOps in a Cloud Native world.
 
-Started 2016 to automate deployments at Weaveworks, the project has grown dramatically since then.
-Especially since the planning of Flux v2 and GitOps Toolkit SDK it was clearer that the scope became
-more ambitious and that the Flux project has become the home for the Flux family of projects,
-all solving specific GitOps needs.
+Started in 2016 to automate deployments at Weaveworks, the project has taken on a new life since then.
+The Flux community has become the home for a family of projects, all solving a variety of specific GitOps needs.
 
 We also want our community to be diverse, helpful, collaborative and a fun place to be, so we would love to have you join us!
 
 ## Getting involved
 
+### Meetings
+
+We run regular meetings and discuss things there.
+
+The upcoming meetings, talks, and community events for the next month are here:
+
+{{< home/calendar >}}
+
+Our calendar data is hosted on the CNCF-Flux-Dev List, which has the option to subscribe: <https://lists.cncf.io/g/cncf-flux-dev/calendar>
+
+We are very happy if new users, contributors and developers join and we can put names to faces!
+
+Review how to [subscribe to the Flux Calendar](#subscribing-to-the-flux-calendar) to add the Flux meetups on your own calendar.
+
+We are looking forward to seeing you there.
+
+#### Recordings
+
+Check out the <https://www.youtube.com/@fluxcd> channel for recordings of Flux Dev Meetings and other Flux-related playlists.
+
+#### Schedules
+
+The recurring meetings and their schedules (in English) are listed below with their source time zones, and a link to any meeting "Minutes" docs.
+
+Some regularly scheduled meetings are scheduled in their hosts' local time zones.
+
+| Which | Times | Agenda & Minutes | Recordings |
+| ----- | ----- | ---------------- | ---------- |
+| Flux Dev Meetings | "early" meeting: Uneven weeks: Wed, 12:00 UTC <br/>"late" meeting: Even weeks: Thu, 15:00 UTC | [Document](https://docs.google.com/document/d/167SKpaDUrpiBqNR2lXnQjXyb5Gxq6uB-Fujz7eBQxyw/edit) | [YouTube](https://www.youtube.com/playlist?list=PLwjBY07V76p5mWNgdINjIiuUiItIeLhIN) |
+| Flux Bug Scrub | "early" meeting: Even weeks: Wed, 8:00 Eastern US time <br/>"late" meeting: Odd weeks: Thu, 13:00 Eastern US time <br/>"AEST Edition" Even weeks: Wed, 8:00 Brisbane time | [Bug Scrub Template (#0090)](https://docs.google.com/spreadsheets/d/1Hv9BIr0vFAOHscqYzmlKKG1Il18Nw8X_iiDyi0HRgCs/edit#gid=2133538565) | [YouTube](https://www.youtube.com/watch?v=c4unRRJp-o0&list=PLwjBY07V76p6J6z30cBRqS_N0Ka6NhEsY) |
+
+Do note that Daylight Savings Time changes can vary across regions, which can cause some confusion.
+
+The calendar at <https://fluxcd.io/community#meetings> has been localized to show the events in your local time zone.
```

Comment:
This link is for the benefit of anyone that happens to be reading this file at https://github.com/fluxcd/community/?tab=readme-ov-file#joining-the-flux-community - since they may not be aware this README.md file is used as an external-source and that the README here actually drives the website page [fluxcd.io/community](fluxcd.io/community).

Probably all of the Flux maintainers who would review this PR already knew that, but...